eng/testing/tests.wasm.targets

```diff
@@ -180,7 +187,8 @@
         OutputFile="$(BundleDir)publish\ProxyProjectForAOTOnHelix.props" />
 
     <Copy SourceFiles="@(BundleFiles)"         DestinationFolder="$(BundleDir)%(TargetDir)" />
-    <Copy SourceFiles="@(_WasmVFSFilesToCopy)" DestinationFiles="$(BundleDir)\extraFiles\%(_WasmVFSFilesToCopy.TargetPath)" />
+    <Copy SourceFiles="@(_WasmVFSFilesToCopy)" DestinationFiles="$(BundleDir)\vfsFiles\%(_WasmVFSFilesToCopy.TargetPath)" />
```

Comment:
This should be there probably only in Debug mode, otherwise the `mono-config.json` will always contain it and the runtime will download it. It would fail if it was missing. (I guess)

I suggest you create WBT for that

Reply:
It won't go into mono-config.json . Adding a test is a good suggestion, thanks 👍